src/status_im/contexts/wallet/common/utils.cljs

```diff
@@ -98,3 +98,8 @@
 (defn calculate-fiat-change
   [fiat-value change-pct-24hour]
   (money/bignumber (* fiat-value (/ change-pct-24hour (+ 100 change-pct-24hour)))))
+
+(def id-to-network
+  {constants/mainnet-chain-id  :ethereum
+   constants/optimism-chain-id :optimism
+   constants/arbitrum-chain-id :arbitrum})
```

Comment:
In Clojure I've seen many times naming these maps as `id->network` the reason is they can be used as:
`(id->network 5)`. So I think it's a more common naming